Panic interface {} is nil, not string

Erroring out, unaware of changes:

```
panic: interface conversion: interface {} is nil, not string

goroutine 1 [running]:
github.com/common-fate/granted/pkg/cfaws.(*Profiles).GetFrecentProfiles(0xc000255980)
	/Users/runner/work/granted-cli-build/granted-cli-build/granted/pkg/cfaws/frecent_profiles.go:59 +0x5d0
github.com/common-fate/granted/pkg/assume.AssumeCommand(0xc0000a3340)
	/Users/runner/work/granted-cli-build/granted-cli-build/granted/pkg/assume/assume.go:216 +0x64c
github.com/urfave/cli/v2.(*Command).Run(0xc0001c3180, 0xc0000a3340, {0xc000034220, 0x1, 0x1})
	/Users/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.24.1/command.go:271 +0xa42
github.com/urfave/cli/v2.(*App).RunContext(0xc000188960, {0xd18690?, 0xc0000420e0}, {0xc000034220, 0x1, 0x1})
	/Users/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.24.1/app.go:333 +0x665
github.com/urfave/cli/v2.(*App).Run(...)
	/Users/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.24.1/app.go:310
main.main()
	/Users/runner/work/granted-cli-build/granted-cli-build/granted/cmd/granted/main.go:50 +0x285
```
Adding more context once I discover.

--- a/pkg/cfaws/frecent_profiles.go
+++ b/pkg/cfaws/frecent_profiles.go
@@ -56,7 +56,11 @@ func (p *Profiles) GetFrecentProfiles() (*FrecentProfiles, []string) {
 
 	// add all frecent profile names in order if they are still present in the profileNames slice
 	for _, entry := range fr.Entries {
-		e := entry.Entry.(string)
+		e, ok := entry.Entry.(string)
+		if !ok {
+			continue
+		}
+
 		if p.HasProfile(e) {
 			names = append(names, e)
 			namesMap[e] = e

